packages/app/src/modules/options/config.js

```diff
@@ -1,18 +1,13 @@
 import {
-    VIS_TYPE_COLUMN,
-    VIS_TYPE_BAR,
-    VIS_TYPE_STACKED_COLUMN,
-    VIS_TYPE_STACKED_BAR,
     VIS_TYPE_PIVOT_TABLE,
-    VIS_TYPE_LINE,
-    VIS_TYPE_RADAR,
-    VIS_TYPE_AREA,
-    VIS_TYPE_STACKED_AREA,
     VIS_TYPE_PIE,
     VIS_TYPE_GAUGE,
-    VIS_TYPE_YEAR_OVER_YEAR_LINE,
-    VIS_TYPE_YEAR_OVER_YEAR_COLUMN,
     VIS_TYPE_SINGLE_VALUE,
+    isStacked as getIsStacked,
```

Comment:
Why the rename?
Perhaps it should be called `isStackedType` for consistency with the others?

Reply:
Because I wanted to pass `isStacked` to `defaultProps` without renaming the prop and obviously I can't have `const isStacked = isStacked(type)` 🙂  alternatively I could keep the original name of the Analytics prop and use a different name for my const, but I thought `isStacked: someOtherName` would be less clean the current way.

Reply:
The exported prop from Analytics could be renamed on the Analytics side but that would cause a breaking change that I don't think is worth it.

Reply:
Yes, I thought about the breaking change.
Let's keep it as it is, or use `isStacked as isStackedType` 😆